### ISSUE TITLE
[$10] [MSFT-70] [Member] Suspicious alternative text is seen in Meet our community Image

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2784]
**Steps to Reproduce**
1. Go to https://accounts.topcoder.com/member
2. Inspect Element
3. Observe the alternative text.

**Expected Result**
alternative text should be present.

**Actual Result**
Suspicious alternative text is found in following image: 

img src="https://www.topcoder.com/wp-content/uploads/2018/04/sample-content-box-1.jpg
https://www.topcoder.com/wp-content/uploads/2018/04/sample-content-box-1.jpg
" alt="box-image">

**Device**: Desktop/Labtop
**Operating System**: macOS
**Browser**: Chrome

**WCAG Category**: 1. Perceivable
**WCAG Level**: A
**WCAG Success Criterion**: 1.1.1 Non-Text Content
**Screen Reader Used**: 
**Accessibility Audit Tool used**: Other

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=ae14bf9a%2D590b%2D4bb4%2Da40a%2D4c6822c880a4&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2FCreeya%2FBug%2070

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096281 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096281 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by asadath1395** [comment]
@veshu We need to change the alt text right?

**Comment by mishacucicea** [comment]
@asadath1395  correct

**Comment by asadath1395** [comment]
@mishacucicea I can't seem to find that page in either repo? Which repo/branch that is present?

**Comment by mishacucicea** [comment]
Sorry this is part of home page which is out of scope for now.

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30097864 has been created for this ticket.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
This ticket was not processed for payment. If you would like to process it for payment, please reopen it, add the ```tcx_FixAccepted``` label, and then close it again<br/><br/>```This is an automated message for crazyk via Topcoder X```
